backend/src/apis/fetch.rs

```diff
@@ -54,7 +54,7 @@ mod tests {
     use super::*;
 
     use crate::contracts::mock::mock_erc20::MockERC20;
-    use crate::contracts::tests::initialize_anvil;
+    use crate::tests::initialize_anvil;
```

Comment:
I think we can remove the `fetch` API as mentioned in issue [#145](https://github.com/summa-dev/summa-solvency/issues/145). Also all the related tests

Reply:
Removed [3d1ab95](https://github.com/summa-dev/summa-solvency/pull/152/commits/3d1ab95db73fcfe38006aa7f300b3a78f15fd678)